charts/rancher-istio/0.0.2/charts/istio-init/files/crd-certmanager-10.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterissuers.certmanager.k8s.io
```

Comment:
I can see CRDs in both `charts/rancher-istio/0.0.2/charts/istio-init/files/crd-certmanager-10.yaml ` and `charts/rancher-istio/0.0.2/templates/crd-certmanager.yaml`. Do we need this replication? The same to Istio CRDs.

Reply:
we will need both, the crd-certmanager.yaml will pre-install CRDs on bootstrap if the crd does not exist, the `istio-init` one is responsible for future apply or update.